client/landing/stepper/declarative-flow/internals/steps-repository/sensei-domain/style.scss

```diff
@@ -3,15 +3,32 @@
 @import "@wordpress/base-styles/variables";
 
 $sensei-mobile-layout-width: 1200px;
+$sensei-color: #43af99;
 
 .sensei.sensei-domain {
-	--color-text-subtle: #a7aaad;
-	--studio-green-60: #1e8133;
-
+	.css-1l7urtg-Container {
+		height: auto;
+	}
```

Comment:
These `.css-1l7urtg-Container` class names are auto generated by styled-components. You cannot rely on them. If you want to add styles to it you need to add them directly in component declaration. [In here](https://github.com/Automattic/wp-calypso/blob/49d7a37764e1f88829be6a6c3fe13783e8060496/packages/onboarding/src/sensei-step-container/components.ts#L3-L12).

Also, what are you trying to achieve? The only difference I noticed is that the progress bar at the top scrolls up with the content when you add this style. Is that a design requirement?

https://user-images.githubusercontent.com/2578542/200493697-2a3824b0-cc13-42b0-9be3-3f363c92ff8c.mp4

If you still want this behavior then you need to add this rule inside the component declaration [here](https://github.com/Automattic/wp-calypso/blob/49d7a37764e1f88829be6a6c3fe13783e8060496/packages/onboarding/src/sensei-step-container/components.ts#L3-L12). Like this
```scss
	.sensei.sensei-domain & {
		height: auto;
	}
```

Reply:
Btw, the reason that container had an explicit `height: 100vh; width: 100%;` is because it makes our step container a `flex` container that covers the whole screen both horizontally and vertically. This makes it very easy for us to position elements the way we want and make it work for all screen sizes.

For example, here in the Sensei Setup step, where user chooses the site title and style, the footer sits at the bottom of the screen no matter the height of the screen.


https://user-images.githubusercontent.com/2578542/200501267-e9a5846c-08c2-4ccd-a942-9dace7c9c95f.mp4